ucp_graphicsApiReplacer/ucp_graphicsApiReplacer/crusaderGraphicsApiReplacer.cpp

```diff
@@ -378,7 +378,7 @@ namespace UCPGraphicsApiReplacer
     d.hasFocus = false;
 
     // found no other way to proper minimize
-    if (confRef.window.type == TYPE_BORDERLESS_FULLSCREEN || confRef.window.type == TYPE_FULLSCREEN)
+    if ((confRef.window.type == TYPE_BORDERLESS_FULLSCREEN || confRef.window.type == TYPE_FULLSCREEN) && confRef.window.minimizeOnLostFocus)

```

Comment:
I would prefer if the setting was general in this case. It appears to apply to everything and I think that is fine, but then it should actually do it. So the condition should only be the flag.

You should however check the codebase via search if there are other places that check for `TYPE_BORDERLESS_FULLSCREEN`, if there is anything else that relates to minimizing. (Although, if there is a case that "maximizes", that I forgot, it probably should stay restricted to fullscreen)

Reply:
Addressed